jsonnet/prometheus-operator/admission-webhook.libsonnet

```
@@ -0,0 +1,123 @@
+local defaults = {
+  local defaults = self,
+  name: 'prometheus-operator-admission-webhook',
+  namespace: error 'must provide namespace',
+  version: error 'must provide version',
+  admissionWebhookImage: error 'must provide admissionWebhookImage',
```

```suggestion
  image: error 'must provide admission webhook image',
```

This is already separated into its own file, so no need to prefix the variable name.